packages/flame_oxygen/lib/src/oxygen_game.dart

```diff
@@ -66,5 +66,6 @@ abstract class OxygenGame extends Game {
 
   @override
   @mustCallSuper
+  //ignore: must_call_super
```

Comment:
Is that right? 🤔

Reply:
It is since Oxygen isn't using the children so it would be an unnecessary check through an empty component tree

Reply:
Hm, I think that is there cases where it makes sense not to call super, we should remove the annotation and explain on the docs, otherwise I believe we should respect it everytime

Reply:
Done